--- pyproject.toml.bkp	2026-06-12 14:46:42
+++ pyproject.toml	2026-06-12 14:46:42
@@ -57,12 +57,12 @@
     "azure-storage-blob==12.28.0",
     "mcp==1.26.0",
     "litellm-proxy-extras==0.4.72",
-    "litellm-enterprise==0.1.41",
+    "litellm-enterprise==0.1.41; platform_machine != 'ppc64le'",
     "RestrictedPython==8.1",
     "rich==13.9.4",
     "polars==1.38.1",
     "soundfile==0.12.1",
-    "pyroscope-io==0.8.16; sys_platform != 'win32'",
+    "pyroscope-io==0.8.16; sys_platform != 'win32' and platform_machine != 'ppc64le'",
     "pydantic-settings>=2.14.1",
 ]
 extra_proxy = [
